projects/storefrontlib/src/layout/main/storefront.component.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('StorefrontComponent', () => {
         DynamicSlotComponent,
         MockPageLayoutComponent,
         MockAsmRootComponent,
-        MockFeatureLevelDirective,
+        MockFeatureDirective,
```

Comment:
difference the two directives? because both exist, but different name only

Reply:
One is for the feature name, the other one is for the level. Before, the `MockFeatureDirective` (the current one) was wrongly named `MockFeatureLevelDirective`. Just a typo :)